keep-ui/app/layout.tsx

```diff
@@ -14,26 +14,27 @@ const mulish = Mulish({
 });
 
 import { ToastContainer } from "react-toastify";
-import Navbar from "./navbar";
+import Navbar from "components/navbar/Navbar";
 
-export default async function RootLayout({
-  children,
-}: {
-  children: React.ReactNode;
-}) {
+type RootLayoutProps = {
+  children: ReactNode;
+};
+
+export default async function RootLayout({ children }: RootLayoutProps) {
   return (
-    <html
-      lang="en"
-      className={`h-full bg-gray-50 ${mulish.className}`}
-      suppressHydrationWarning={true}
-    >
-      <body className="h-full">
+    <html lang="en" className={`bg-gray-50 ${mulish.className}`}>
+      <body className="h-screen grid grid-cols-[250px_auto]">
         <NextAuthProvider>
+          {/* @ts-expect-error Server Component */}
```

Comment:
can you leave a comment on why do we need this?

Reply:
Resolved with https://github.com/keephq/keep/pull/825/commits/32793c29f6c58ae7037c0b9bf99ce786e1a0222a
Apparently, either the React or NextJS team pushed an update to allow JSX promises (for React Server Components)